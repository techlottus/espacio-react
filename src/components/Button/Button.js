

import React, { useEffect } from 'react'


export const Button = React.memo(({ data, onClick }) => {



  const buttonRef = React.createRef();
  
  useEffect(() => {
    buttonRef.current.data = {
      id: data.id || null,
      type: data.type  || 'primary',
      title: data.title || '',
      size: data.size || 'small',
      icon: data.icon || '',
      lyIcon: data.lyIcon,
      disabled: data.disabled,
      isExpand: data.isExpand,
      test: data.test || ''
    }
  }, [data]) // eslint-disable-line react-hooks/exhaustive-deps

  useEffect(() => {
    buttonRef.current.addEventListener('onClick', onClick);
  }, []) // eslint-disable-line react-hooks/exhaustive-deps

  return (
    <>
      <lottus-button ref={buttonRef}></lottus-button>
    </>
  )
})


