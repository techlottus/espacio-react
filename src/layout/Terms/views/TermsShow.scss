@import "../../../App.scss";

.terms-page{

  .terms-page-container {
    padding: pixeltorem(90px) pixeltorem(5px) 0  pixeltorem(5px);
    margin: 0 pixeltorem(15px);
    font-family: "Nunito", sans-serif; 

    @extend .fixedcontainer-height;

    @media only screen and (min-width: map-get($grid-breakpoints, m)) {
      margin: 0 pixeltorem(35px) 0 pixeltorem(60px);
      font-family: "Nunito", sans-serif;
      padding-top: pixeltorem(45px);
      height: 100%;
    }

    .terms-arrow {
      display: block;
      padding-bottom: pixeltorem(20px);
      color: palette-colors($brand, p00);

      @media only screen and (min-width: map-get($grid-breakpoints, m)) {
        display: none; 
      }
    }
  
    .terms-content {
      max-height: pixeltorem(320px);
      padding-right: pixeltorem(40px);
      overflow-y: scroll;
      > .termshow-title {
        font-family: 'Bebas',sans-serif;
        font-size: pixeltorem(24px);
        padding-bottom: pixeltorem(16px);

        @media only screen and (min-width: map-get($grid-breakpoints, m)) {
          font-size: pixeltorem(56px);
        }
      }
      .terms-map {
        > .termshow-subtitle {
          padding: pixeltorem(5px) 0;
          font-size: pixeltorem(16px);
          font-weight: 700;
        }
  
        .termshow-texts{
           > .termshow-text {
            font-size: pixeltorem(12px);
            padding: pixeltorem(10px) 0;
          }
    
          > .termshow-ul {
            .termshow-li {
              padding: pixeltorem(5px) 0;
              font-size: pixeltorem(12px);
    
              &::before {
                content: "\2022";
                color: map-get($palette-neutral, n80);
                font-weight: bold;
                display: inline-block;
                width: 1em;
                margin-left: 1em;
              }
            }
          }
          > .termshow {
            padding: pixeltorem(5px) 0;
            font-size: pixeltorem(12px);  
          }
          .termshow-ol-number {
            .termshow-li-number {
              padding: pixeltorem(5px) 0;
              font-size: pixeltorem(12px);
              list-style-type: auto;
              margin-left: 4em;
            }
          }
           .termshow-ol-upperroman {
            .termshow-li-upperroman {
              padding: pixeltorem(5px) 0;
              font-size: pixeltorem(12px);
              list-style-type: upper-roman;
              margin-left: 5em;
            }
          }
          .termshow-ul-circle {
            .termshow-li-circle {
              padding: pixeltorem(5px) 0;
              font-size: pixeltorem(12px);
              list-style-type: circle;
              margin-left: 5em;
            }
          }
          .termshow-ul-square {
            .termshow-li-square {
              padding: pixeltorem(5px) 0;
              font-size: pixeltorem(12px);
              list-style-type: square;
              margin-left: 5em;
            }
          }
          .termshow-ul-guion {
            .termshow-li-guion {
              padding: pixeltorem(5px) 0;
              font-size: pixeltorem(12px);
              margin-left: 3em;

              &::before {
                content: "-";
                margin-right: 5px;
              }
            }
          }
        }
      }
    }
  
    .terms-feed {
      padding-top: pixeltorem(20px);

      @media only screen and (min-width: map-get($grid-breakpoints, m)) {
        padding-top: pixeltorem(60px);
      }
    }
  
    .terms-btns {
      padding: pixeltorem(20px) 0;

      @media only screen and (min-width: map-get($grid-breakpoints, m)) {
        display: flex;
        justify-content: flex-end;
      }

      .button-wrap {
        padding: pixeltorem(8px) 0;

        @media only screen and (min-width: map-get($grid-breakpoints, m)) {
          padding: 0 pixeltorem(10px);
        }

        button{
          width: 100%;
          text-align: center;
          justify-content: center;
        }
      }
    }
  }
}