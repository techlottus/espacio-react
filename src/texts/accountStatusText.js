export const accountStatusText = {
  statusTitle: "Estado de cuenta y consultas de facturas",
  resumeTitle: "Resumen de adeudo",
  total: "Total",
  errorMessage:
    "Tuvimos dificultades para cargar tu información en este momento",
  emptyMessage: {
    congrats: "Felicidades, estás al día con tus pagos",
    continue: "Continúa transformando tu futuro",
  },
  images: { 
    user: "images/user.png",
    empty: "yeah.png",
  },
  btnModal: {
    delete: "Eliminar",
    cancel: "Cancelar",
  },
  btnDownload: {
    download: "Descargar Estado de cuenta",
  },
  btnInvoice: {
    checkInvoice: "Consulta tus facturas",
  },
  tableAccountStatus: [
    { title: "Clave" },
    { title: "Ciclo" },
    { title: "Periodo" },
    { title: "Adeudo" },
  ],

  periodDetailTitle: "Detalles por periodo",
  periodDetailDescription:
    "Selecciona el programa y el periodo para ver los detalles de los movimientos; se clasifican de la siguiente manera:",
  periodDetailBullets: {
    debt: "Con adeudo",
    payed: "Pagados",
    toPay: "Por pagar",
  },
  periodDetailSelects: {
    program: "Programa",
    period: "Periodo",
  },
  accountDetailPeriod: {
    title: "Periodo",
    icon: "calendar_today",
    debt: "Saldo del periodo",
  },
  accountDetailMovements: {
    title: "Movimientos",
  },
  tableInvoiceQuery: [
    { title: "Fecha de emisión" },
    { title: "Periodo" },
    { title: "Concepto" },
    { title: "Monto" },
    { title: "RFC" },
    { title: "Preview" },
    { title: "PDF" },
    { title: "XML" },
  ],
  tableRowInvoiceQuery: [
    { title: "07 Abril 2021" },
    { title: "Periodo 5" },
    { title: "Colegiatura" },
    { title: "$2405.50" },
    { title: "LRKO485767HF2" },
  ],
  accountStatusInvoiceQuery: {
    title: "Consulta de facturas",
    description:
      "Filtra por rango de fechas, selecciona el perido que deseas consultar y presiona descargar.",
  },
  btnInvoiceQuery: {
    download: "Descargar todo",
    clear: "Limpiar",
  },
  linkInvoiceQuery: "Ver más"
};
