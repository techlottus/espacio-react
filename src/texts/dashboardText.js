export const dashboardText = {
  menu: {
    aula: {
      emoji: "mortar-board",
      text: "Aula",
    },
    book: {
      emoji: "books",
      text: "Biblioteca",
    },
    helpCenter: {
      emoji: "thought-balloon",
      text: "Centro de Ayuda",
    },
    procedures: {
      emoji: "file-folder",
      text: "Trámites",
    },
    accountStatus: {
      emoji: "chart-with-upwards-trend",
      text: "Estado de Cuenta",
    },
    payment:  {
      emoji: "credit-card",
      text: "Pagos",
    },
    invoice: {
      emoji: "page-facing-up",
      text: "Facturas"
    },
    benefits: {
      emoji: "star",
      text: "Portal de Beneficios"
    },
    sede: {
      emoji: "notebook",
      text: "SeDe"
    },
    psychopedagogical: {
      emoji: "blue-heart",
      text: "Centro de Atención Psicopedagógica"
    },
    evaluateServices: {
      emoji: "ec ec-white-check-mark",
      text: "Evalúa los servicios"
    },
    registrationSubjects: {
      emoji: "ec ec-closed-book",
      text: "Inscribe tus materias"
    },
    logout: {
      emoji: "back",
      text: "Cerrar sesión",
    },
  },
  titleWelcome: "Hola",
  images: {
    cardHeader: "images/header.png",
    empty: "empty.png",
  },
  logo: {
    urlLogo: "logo.svg"
  },
  logos: "logo.svg",
  sectionDashboard: {
    titleNews: "Noticias de tu universidad",
    titleCards: "¿Qué quieres hacer hoy?",
  },
  playIcon: "play_circle_outline",
  cards: {
    one: {
      title: "Aula",
      icon: "school",
    },
    two: {
      title: "Pagos",
      icon: "credit_card",
    },
    three: {
      title: "Biblioteca",
      icon: "import_contacts",
    },
    four: {
      title: "Trámites",
      icon: "post_add",
    },
    five: {
      title: "Inscribe tus materias",
      icon: "add_circle_outline",
    },
  },
  footerText: {
    text: "©2022 Derechos reservados Lottus Education",
    privacy: "Aviso de privacidad y términos y condiciones",
  },
};