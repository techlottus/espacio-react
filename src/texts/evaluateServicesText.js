export const evaluateServicesText = {
    evaluateServicesTitle: "Evalua tus servicios digitales",
    modalSurvey: {
        title: "Evalua tu experiencia para realizar tu pago a través de Espacio UTC",
        text: "¿Que tan probable es que recomiendes los pagos en Espacio UTC?"
    }
}