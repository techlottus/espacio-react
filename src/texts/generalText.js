export const generalText = {
  modals: {
    modalExit: {
      icon: "help",
      modalTitlePayment: "¿Seguro que deseas salir de la sección de pagos?",
      modalTitleProcedure: "¿Seguro que deseas abandonar el trámite?",
      exitBtn: "Aceptar",
      cancelBtn: "Cancelar",
    },
  },
};
