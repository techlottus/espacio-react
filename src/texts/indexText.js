import { termsText } from "./termsText";
import { paymentText } from "./paymentText";
import { dashboardText } from "./dashboardText";
import { accountStatusText } from "./accountStatusText";
import { loginText } from "./loginText";
import { screenText } from "./screenText";
import { sidebarText } from "./sidebarText";
import { proceduresText } from "./proceduresText";
import { generalText } from "./generalText";
import { helpCenterText } from "./helpCenterText";
import { evaluateServicesText } from "./evaluateServicesText";
import { registrationSubjectsText } from "./registrationSubjectsText";

export const texts = {
  terms: termsText,
  payment: paymentText,
  dashboard: dashboardText,
  accountStatus: accountStatusText,
  login: loginText,
  screen: screenText,
  sidebar: sidebarText,
  procedures: proceduresText,
  general: generalText,
  helpCenter: helpCenterText,
  evaluateServices: evaluateServicesText,
  registrationSubjects: registrationSubjectsText
}