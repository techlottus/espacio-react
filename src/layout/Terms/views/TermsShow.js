import React, { useState } from "react";
import Notification from "../views/Notification";
import { Button } from "../../../components/Button/Button";
import { termsBtns } from "../../../constants/Terms.constant";

import "./TermsShow.scss";
import { Link } from "react-router-dom";
import { texts } from "../../../texts/indexText";
import { sendInfoTM } from "../../../tagging/services/sendInfoTagManager";
import {
  privacyBtnDeclinedTermsTag,
  termsBtnDeclinedTermsTag,
} from "../../../tagging/flows/termsTag";
import { testTermsTitle } from "../../../constants/test/testTerms";

const TermsShow = ({ title, contents, clickShow, type }) => {
  const [btns] = useState([...termsBtns]);

  return (
    <div className="terms-page">
      <div className="terms-page-container">
        <Link to="/terms">
          <div className="terms-arrow">
            <span className="material-icons">arrow_back</span>
          </div>
        </Link>
        <div className="terms-content">
          <h2 className="termshow-title">{title}</h2>
          {contents.map((content, i) => {
            return (
              <div className="terms-map" key={i}>
                <h3 className="termshow-subtitle">{content.subtitle}</h3>
                <div
                  className="termshow-texts"
                  dangerouslySetInnerHTML={{ __html: content.text }}
                ></div>
              </div>
            );
          })}
        </div>
        <div className="terms-feed">
          <Notification icon={"info"} text={texts?.terms.notification} />
        </div>
        <div className="terms-btns">
          {btns.map((data, index) => {
            return (
              <div className="button-wrap" key={index}>
                <Button
                  data={{
                    ...data,
                    test: testTermsTitle.btnTermsAndPrivacyAction + '-' +type + '-' +data.id
                  }}
                  onClick={() => {
                    clickShow(data.id, type);
                    type === "privacy" && data.id === "decline"
                      ? sendInfoTM(
                          window,
                          privacyBtnDeclinedTermsTag,
                          "noacepto"
                        )
                      : sendInfoTM(
                          window,
                          privacyBtnDeclinedTermsTag,
                          "acepto"
                        );
                    type === "terms" && data.id === "decline"
                      ? sendInfoTM(window, termsBtnDeclinedTermsTag, "noacepto")
                      : sendInfoTM(window, termsBtnDeclinedTermsTag, "acepto");
                  }}
                />
              </div>
            );
          })}
        </div>
      </div>
    </div>
  );
};

export default TermsShow;
