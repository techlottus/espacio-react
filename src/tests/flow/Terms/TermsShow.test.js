import React from "react";
import { mount } from "enzyme";
import { Provider } from "react-redux";
import "@testing-library/jest-dom";
import configureStore from "redux-mock-store";
import thunk from "redux-thunk";
import TermsShow from "../../../layout/Terms/views/TermsShow";
import { texts } from "../../../texts/indexText";
import { BrowserRouter } from "react-router-dom";
import { act } from "react-dom/test-utils";
import { Button } from "../../../components/Button/Button";
import { termsText } from "../../../texts/termsText";

const middlewares = [thunk];
const mockStore = configureStore(middlewares);
window.dataLayer = {
  push: jest.fn()
}

let initState = {
  terms: {},
  texts: {
    terms: {
      ...termsText
    }
  }
};
let store = mockStore(initState);
const privacyContent = texts?.terms.privacyContent;

const handleMount = (initState) =>  {
  store = mockStore(initState)
  return mount(
    <Provider store={store}>
    <BrowserRouter>
      <TermsShow clickShow={jest.fn()} contents={privacyContent.contents} />
    </BrowserRouter>
  </Provider>
  )
}


describe("Test <TermsShow/> ", () => {
  beforeEach(() => {
  });

  test("init component", () => {
    const wrapper = handleMount(initState)
    expect(wrapper).toMatchSnapshot();
  });

  describe('Should accept or decline', () => {
    test ('should decline', () => {
      const wrapper = handleMount(initState);
  
      act(() => {
        wrapper.find(Button).first().props().onClick()
      })
    })

    test ('should accept', () => {
      const wrapper = handleMount(initState);
  
      act(() => {
        wrapper.find(Button).last().props().onClick()
      })
    })
  })


});
