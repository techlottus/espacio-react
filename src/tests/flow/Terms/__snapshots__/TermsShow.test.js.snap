// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test <TermsShow/>  init component 1`] = `
<div
  className="terms-page"
>
  <div
    className="terms-page-container"
  >
    <a
      href="/terms"
      onClick={[Function]}
    >
      <div
        className="terms-arrow"
      >
        <span
          className="material-icons"
        >
          arrow_back
        </span>
      </div>
    </a>
    <div
      className="terms-content"
    >
      <h2
        className="termshow-title"
      />
      <div
        className="terms-map"
        key="0"
      >
        <h3
          className="termshow-subtitle"
        >
          Identidad y domicilio del responsable
        </h3>
        <div
          className="termshow-texts"
          dangerouslySetInnerHTML={
            Object {
              "__html": "<p class=\\"termshow-text\\"> El Centro Educativo Tres Culturas, S.C. y la Universidad de las Tres Culturas, S (en lo sucesivo conjuntamente la “UTC”), con domicilio en Calle Londres 71-PA, colonia Juárez Alcaldía Cuauhtémoc, C.P. 06600, Ciudad de México, son sociedades constituidas de conformidad con las leyes de los Estados Unidos Mexicanos, ofreciendo sus servicios en todo el territorio nacional comprometidas con la protección de sus Datos Personales, cuando son responsables de su uso, manejo y confidencialidad; por tanto, es importante informarle que los Datos Personales que UTC recabe sobre Usted, son necesarios para verificar y confirmar su identidad, integrar los expedientes para administrar y operar los servicios que presta, a efecto de cumplir con el objeto de UTC de conformidad con lo establecido en la Ley Federal de Protección de Datos Personales en Posesión de Particulares (la “Ley de Datos Personales”).</p> <p class=\\"termshow-text\\"> El Centro Educativo Tres Culturas, S.C. y la Universidad de las Tres Culturas, S.C. (en lo sucesivo conjuntamente la “UTC”), con domicilio en Calle Londres 71-PA, colonia Juárez, Alcaldía Cuauhtémoc, C.P. 06600, Ciudad de México, son sociedades constituidas de conformidad con las leyes de los Estados Unidos Mexicanos, ofreciendo sus servicios en todo el territorio nacional, y comprometidas con la protección de sus Datos Personales, cuando son responsables de su uso, manejo y confidencialidad; por tanto, es importante informarle que los Datos Personales que UTC recabe sobre Usted, son necesarios para verificar y confirmar su identidad, integrar los expedientes para administrar y operar los servicios que presta, a efecto de cumplir con el objeto de UTC de conformidad con lo establecido en la Ley Federal de Protección de Datos Personales en Posesión de Particulares (la “Ley de Datos Personales”).</p> <p class=\\"termshow-text\\"> El Centro Educativo Tres Culturas, S.C. y la Universidad de las Tres Culturas, S.C. (en lo sucesivo conjuntamente la “UTC”), con domicilio en Calle Londres 71-PA, colonia Juárez, Alcaldía Cuauhtémoc, C.P. 06600, Ciudad de México, son sociedades constituidas de conformidad con las leyes de los Estados Unidos Mexicanos, ofreciendo sus servicios en todo el territorio nacional, y comprometidas con la protección de sus Datos Personales, cuando son responsables de su uso, manejo y confidencialidad; por tanto, es importante informarle que los Datos Personales que UTC recabe sobre Usted, son necesarios para verificar y confirmar su identidad, integrar los expedientes para administrar y operar los servicios que presta, a efecto de cumplir con el objeto de UTC de conformidad con lo establecido en la Ley Federal de Protección de Datos Personales en Posesión de Particulares (la “Ley de Datos Personales”). </p>",
            }
          }
        />
      </div>
      <div
        className="terms-map"
        key="1"
      >
        <h3
          className="termshow-subtitle"
        >
          Datos personales que recaba la UTC
        </h3>
        <div
          className="termshow-texts"
          dangerouslySetInnerHTML={
            Object {
              "__html": "<p class='termshow-text'>Son los datos de identificación, laborales, financieros o académicos que usted nos proporcione de forma directa, ya sea personalmente o por medios electrónicos, ópticos, sonoros, visuales o a través de cualquier otra tecnología como internet o vía telefónica, o que usted publique mediante las redes sociales, obteniendo su consentimiento expreso, por escrito, vía electrónica y/o cualquier otro medio de autenticación, para recabarlos, tratarlos y/o transferirlos en términos de la Ley de Datos Personales. La UTC no trata datos personales sensibles. </p><p class='termshow-text'> La UTC recaba y da tratamiento a los siguientes datos personales de manera lícita, siempre y cuando usted no manifieste objeción alguna en el momento que nos proporciona sus datos personales directamente o manifieste su oposición posterior por los medios descritos en el presente aviso.</p><p class='termshow-text'> Se consideran datos personales el nombre (s) y apellido, edad, fecha de nacimiento, nacionalidad, domicilio, dirección de correo electrónico, nombre de identificación en redes sociales, número telefónico, estudios previos y datos personales patrimoniales como nivel de ingresos, datos laborales de contacto de los empleadores, dependientes económicos y demás de naturaleza análoga.</p>",
            }
          }
        />
      </div>
      <div
        className="terms-map"
        key="2"
      >
        <h3
          className="termshow-subtitle"
        >
          Finalidades
        </h3>
        <div
          className="termshow-texts"
          dangerouslySetInnerHTML={
            Object {
              "__html": "<p class='termshow-text'>La UTC tratará sus datos personales para las siguientes finalidades que hayan dado origen y sean necesarias para la relación jurídica que tengamos con usted: dar cumplimiento a obligaciones, facturación y cobranza, determinar su admisibilidad e inscribirlo al programa educativo que usted haya seleccionado, impartir nuestros programas académicos, proveer servicios o productos requeridos y registro y gestión de servicios o aplicaciones que se ofrecen en nuestro sitio de internet.</p> <p class='termshow-text'>La UTC también podrá utilizar sus datos para las siguientes finalidades que no son necesarias para la relación jurídica antes mencionada pero que nos permiten y facilitan brindarle una mejor atención: informar sobre cambios o nuevos programas o servicios que puedan ser de su interés, esfuerzos de mercadotecnia o publicidad, evaluar la calidad de nuestros programas o servicios, realizar estudios internos sobre hábitos de consumo, fomentar la comunicación y colaboración entre miembros de la comunidad de la UTC a través de redes sociales y para analizar cómo se utilizan nuestros servicios y nuestro sitio de internet, así como para mejorarlos. Si usted no desea que la UTC trate sus datos personales para estos fines adicionales, por favor envíe un correo electrónico a datospersonales@utc.mx.</p> <p class='termshow-text'>Adicionalmente, utilizaremos sus datos para las finalidades siguientes:</p><ul class='termshow-ul'><li class='termshow-li'>Realizar estudios internos y externos estadísticos sobre los servicios educativos y actividades deportivas y culturales desarrollados en la UTC.</li><li class='termshow-li'>Promoción de servicios ofrecidos por la UTC y sus empresas relacionadas.</li>      <li class='termshow-li'>Seguimiento y atención de alumnos.</li>      <li class='termshow-li'> Atención y servicios a egresados.</li>      <li class='termshow-li'>Encuestas y evaluaciones de servicios educativos o productos.</li>     <li class='termshow-li'> Aquellos relacionados con la identificación, operación, administración del expediente de alumnos y egresados, así como aquellos procesos análogos que sean necesarios para la prestación de los servicios.</li>    <li class='termshow-li'>  Obtener de los empleadores la opinión de los alumnos egresados y que formen parte de la fuerza de trabajo del empleador, respecto de la calidad académica de los egresados, de su desempeño y de las necesidades de egresado profesional de la institución.</li>     <li class='termshow-li'> Contar con un expediente académico y/o administrativo, así como la creación de perfil del alumno y/o colaborador.</li> </ul>",
            }
          }
        />
      </div>
      <div
        className="terms-map"
        key="3"
      >
        <h3
          className="termshow-subtitle"
        >
          Protección de los Datos Personales
        </h3>
        <div
          className="termshow-texts"
          dangerouslySetInnerHTML={
            Object {
              "__html": "<p class='termshow-text'>Para cumplir con las finalidades previstas en el presente aviso, los Datos Personales serán recabados y tratados bajo estrictas medidas de seguridad, y confidencialidad. Por ello, Los Datos Personales que sean proporcionados, que actualmente o en el futuro obren en posesión de UTC, serán manejados, tratados y/o utilizados bajo los principios de licitud, consentimiento, calidad, información, proporcionalidad y responsabilidad, por lo que UTC se compromete a observar y cumplir los citados principios, así como a no vender, alquilar, compartir o divulgar su información personal con fines ilícitos o contrarios a los fines para lo cual fueron proporcionados, salvo los casos excepcionales señalados en la Ley de Datos Personales y su respectivo Reglamento.</p><p class='termshow-text'>UTC realizará sus mejores esfuerzos para proteger los Datos Personales y salvaguardar la seguridad en los sistemas utilizados para almacenar los Datos Personales.</p>     <p class='termshow-text'> Sin embargo, UTC no asume ninguna obligación de mantener confidencial cualquier otra información que el titular proporcione de manera adicional a la solicitada, incluyendo aquella información que el titular facilite a través formularios en el Sitio web.</p>      <p class='termshow-text'>Los Datos Personales que proporcione son almacenados en servidores alojados en instalaciones seguras de UTC.</p>      <p class='termshow-text'>Con la finalidad de asistirlo de forma más eficaz, todas las transacciones en línea son administradas mediante un servidor seguro, el cual recibe la información necesaria para verificar y autorizar el cargo a su método de pago; transferencias interbancarias, tarjetas bancarias y/o depósitos en tiendas de conveniencia, para así procesar y enviar su transacción. Dichas transacciones relacionadas con sus operaciones en línea son administradas bajo altos estándares de seguridad y protección.</p>",
            }
          }
        />
      </div>
      <div
        className="terms-map"
        key="4"
      >
        <h3
          className="termshow-subtitle"
        >
          Transferencias
        </h3>
        <div
          className="termshow-texts"
          dangerouslySetInnerHTML={
            Object {
              "__html": "<p class='termshow-text'>Nos comprometemos a no transferir su información personal a terceros sin su consentimiento, salvo las excepciones previstas en el artículo 37 de la Ley Federal de Protección de Datos Personales en Posesión de los Particulares, así como a realizar esta transferencia en los términos que fija la normatividad aplicable.</p>     <p class='termshow-text'> Por medio del presente se informa que, UTC es parte de un grupo corporativo propiedad de Lottus Education, S.A.P.I. de C.V., con diversas empresas subsidiarias y afiliadas, por lo que, sus datos pueden ser compartidos con nuestras empresas afiliadas, filiales o subsidiarias así como con terceros prestadores de servicios, lo cual en todo momento se hará conforme a la normatividad aplicable, además dichos datos también podrán ser compartidos a un tercero en caso de reorganización, venta, fusión, empresa conjunta, cesión, transferencia u otra disposición de todo o parte de nuestro negocio, activos o acciones, incluso en el caso de reestructuración corporativa o para prospectos de inversionistas, mismos que, en todo momento, estarán obligados a tratar dichos datos de manera confidencial y a no aprovecharse de ellos sin el consentimiento de sus respectivos titulares.</p>     <p class='termshow-text'> En este sentido los datos personales anteriormente descritos podrán ser transferidos y tratados por los terceros abajo descritos. Todas las finalidades mencionadas están en los supuestos del art. 37 de la ley, por lo que no es necesario obtener el consentimiento para su transferencia:</p>      <ul class='termshow-ul'><li class='termshow-li'>Bancos: Realizar pagos y/o devoluciones.</li>     <li class='termshow-li'> Medios de comunicación: Promoción y cumplimiento de eventos y sorteos.</li><li class='termshow-li'>Aseguradoras: Tramite de seguros y garantías.</li><li class='termshow-li'>Despachos: Consultoría jurídica y gestión de cobranza.</li><li class='termshow-li'>Dependencias gubernamentales: Cumplimiento de la legislación aplicable.</li>      <li class='termshow-li'>Firmas de servicios profesionales: Auditoria, encuestas a alumnos y egresados, consultoría y soporte</li> <li class='termshow-li'>Entidades certificadoras: Validación en el cumplimiento de mejores prácticas.</li> </ul>",
            }
          }
        />
      </div>
    </div>
    <div
      className="terms-feed"
    >
      <div
        className="notification-container"
      >
        <div
          className="side -normal"
        >
          <span
            className="material-icons icons"
          >
            info
          </span>
        </div>
        <div
          className="content"
        >
          <p
            className="content-text null"
          >
            Si tienes dudas sobre los documentos, comunícate con tu asesor para que te brinde mayor información
          </p>
        </div>
        <div
          className="side"
        />
      </div>
    </div>
    <div
      className="terms-btns"
    >
      <div
        className="button-wrap"
        key="0"
      >
        <lottus-button />
      </div>
      <div
        className="button-wrap"
        key="1"
      >
        <lottus-button />
      </div>
    </div>
  </div>
</div>
`;
